unittest/memtracer/CMakeLists.txt

```diff
@@ -8,5 +8,6 @@ FOREACH (F ${ALL_SRC})
     MESSAGE("Build ${prjName} according to ${F}")
     ADD_EXECUTABLE(${prjName} ${F})
     TARGET_LINK_LIBRARIES(${prjName} pthread dl GTest::gtest_main memtracer)
+    set_target_properties(${prjName} PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${UNITTEST_OUTPUT_PATH})
```

Comment:
actually this is no need? because `SET(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${UNITTEST_OUTPUT_PATH})` in topper cmakelists.txt?